R/bookdown/github_publish.txt

```diff
@@ -0,0 +1,6 @@
+1. Create the site and make sure that it builds locally
+2. Run `rsconnect::writeManifest(contentCategory = "site")` to create a manifest.json that has the right content_category type, ("content_category": "site",)
+3. Commit and push the site to Github
+4. Login to Connect --> Publish --> Import from Git (use HTTPS git repo URL)
```

Comment:
does this have screenshots somewhere of what this looks like (eg where the Publish button is?) eg something like 
![image](https://user-images.githubusercontent.com/3196313/195409461-821946af-ff13-4fa3-bcbd-882defeda4a5.png)

Reply:
This text is not intended for publication.  See note that I inserted.